python/feathub/processors/flink/flink_feature_table.py

```diff
@@ -0,0 +1,124 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import uuid
+from datetime import timedelta
+from typing import Dict, Any, Optional, List
+
+from feathub.feature_tables.feature_table import FeatureTable
+from feathub.table.schema import Schema
+
+
+class FlinkConnectorSource(FeatureTable):
```

Comment:
Does this mean that some source/slink in FeatHub can only be interpreted by Flink processor? This will break the current FeatHub design where user's FeatHub program (e.g. their source/sink definition) can be interpreted by any processor (e.g. local processor, Flink processor).